.github/workflows/release-to-prod.yml

```diff
@@ -0,0 +1,33 @@
+name: Release to Production
+on:
+  pull_request:
+    branches:
+      - production
```

Comment:
Why using the production branch? IMHO using branches for environments in an Open Source repository isn't a good pattern. Our repository hosts the code of our application, and to know what's in production we can know by checking the latest released tag.

We have multiple instances of Parabol (PPMIs and both SaaS) and we might have more in the future. IMHO it doesn't scale. 

If we are going to do this, I would at least use a branch public-saas instead of production. At least it would be clearer.

Reply:
oh man, i didn't even think about pushing to PPMIs from github! I say we keep PPMIs out of github, since devs shouldn't even have to mess with them. i don't mind the name, but we call it production everywhere else, so i figured we could keep the name here, too?

Reply:
Let's keep production then. If we need to change it in the future, we can change it easily.